client/src/pages/Home.js

```diff
@@ -51,12 +57,14 @@ export function Home() {
 
 				<div className="icons">
 					{dataList.map((dataItem) => (
+						<div key={dataItem.id}>
 						<CityIcon
 							cityName={dataItem.location}
-							onClick={() => handleCityClick(dataItem.location)}
-							key={dataItem.id}
+							onClick={() => handleShow(dataItem)}
 						/>
+						</div>
 					))}
+					<ModalVideo show={show} handleClose={handleClose} title={selectedIcon ? selectedIcon.title:""} url={selectedIcon ?selectedIcon.url: ""} location={selectedIcon ? selectedIcon.location:""} />
```

Comment:
Normally linting will take care of this, but I recommend moving these onto separate lines for readability, e.g.
```jsx
<MyComponent
  show={show}
  otherProp={otherProp} 
  propWithLogic={condition ? true : false} />
```